kafka/assets/dashboards/kafka_dashboard.json

```diff
@@ -139,34 +113,23 @@
                       "value": 10
                     }
                   ],
-                  "formulas": [
-                    {
-                      "formula": "query1"
-                    }
-                  ],
+                  "formulas": [{ "formula": "query1" }],
                   "queries": [
                     {
                       "aggregator": "avg",
                       "data_source": "metrics",
                       "name": "query1",
-                      "query": "sum:kafka.replication.under_replicated_partitions{$datacenter}.weighted()"
+                      "query": "sum:kafka.replication.under_replicated_partitions{$env}.weighted()"
```

Comment:
datacenter isn't a standard tag in orgs (this is Org2's selected secondary primary tag), env is